frontend/util.lua

```diff
@@ -1415,6 +1415,8 @@ end
 -- @boolean case_sensitive
 -- @number start_pos Position number in text to start search from
 -- @treturn number Position number or 0 if not found
+-- @treturn table Text char list
+-- @treturn table Search string char list
 function util.stringSearch(txt, str, case_sensitive, start_pos)
```

Comment:
Feels a little odd that this function in `util` would return some of its internal stuff.
I know it costs nothing (even if the caller does not use and store them, they will soon be GC'ed), and it's a good optimisation for TextViewer so it doesn't have to re-do it again, but it still feels a little odd and ad-hoc.
(Alternatives that don't feel better: duplicate it with this extended return values in TextViewer. Or add a param `return_charlists=true` so it still looks plain as before when not provided.)